pandas/core/ops.py

```diff
@@ -582,6 +642,14 @@ def _get_op_name(op, special):
 DataFrame.{reverse}
 """
 
+_flex_comp_doc_FRAME = """
+Wrapper for flexible comparison methods {name}
```

Comment:
This could use a description of what “flexible” means in this context.

Reply:
We maintained original text and only added examples. Please advise on text changes.